README.md

```diff
@@ -0,0 +1,226 @@
+}
+```
+
+> **Note:** `UseOpenIddict()` must be used ***after*** `app.UseIdentity()` and any external providers.
+
+### Configuration & Options.
+
+OpenIddict have multiple options you can set to customize for your 
+requirements/needs.
+
+<table>
+<thead>
+    <tr><th>Options</th><th>type</th><th>Description</th><th>Default</th></tr>
+</thead>
+<tbody>
+<tr>
+<td>AuthenticationScheme</td>
+<td>string</td>
+<td>
+    The base address used to uniquely identify the authorization server.<br>
+    The URI must be absolute and may contain a path, but no query string or fragment part.<br>
+    Unless <code>AllowInsecureHttp</code> has been set to <code>true</code>, an HTTPS address must be provided.
+</td>
+<td><code>oidc-server</code></td>
+</tr>
+<tr>
+  <td>Issuer</td>
+  <td>Uri</td>
+  <td>The base address (absolute) used to uniquely identify the authorization server.<br>
+      <sub><i>Unless <code>AllowInsecureHttp</code> has been set to <code>true</code>, an HTTPS address must be provided.</i></sub>
+  </td>
+  <td>Automatically inferred from the request URL</td>
+</tr>
+<tr>
+  <td>
+    AuthorizationEndpointPath
+    <br>
+    LogoutEndpointPath
+  </td>
+  <td>PathString</td>
+  <td>The path of the endpoint. Can be set to <code>PathString.Empty</code> to disable the endpoint.</td>
+  <td>
+      <code>/connect/authorize</code>
+      <br>
+      <code>/connect/logout</code>
+  </td>
+      
+</tr>
+<tr>
+  <td>
+    AuthorizationCodeLifetime
+    <br>
+    AccessTokenLifetime
+    <br>
+    IdentityTokenLifetime
+    <br>
+    RefreshTokenLifetime
+  </td>
+  <td>TimeSpan</td>
+  <td>The period of time the token or code remains valid after being issued.</td>
+  <td>
+    5 minutes
+    <br>
+    1 hour
+    <br>
+    20 minutes
+    <br>
+    6 hours
+  </td>
+</tr>
+<tr>
+  <td>UseSlidingExpiration</td>
+  <td>bool</td>
+  <td>
+    Determines whether refresh tokens issued during a <code>grant_type=refresh_token</code> request should be generated with a new expiration date or should re-use the same expiration date as the original refresh token.
+    <br>
+    <sub>Set this property to <code>true</code> to assign a new expiration date each time a refresh token is issued, <code>false</code> to use the expiration date of the original refresh token.</sub>
+  </td>
+  <td><code>true</code></td>
+</tr>
+<tr>
+  <td>ApplicationCanDisplayErrors</td>
+  <td>bool</td>
+  <td>Set to <code>true</code> if the web application is able to render error messages on the authorization endpoint.</td>
+  <td><code>false</code></td>
+</tr>
+<tr>
+  <td>AllowInsecureHttp</td>
+  <td>bool</td>
+  <td>
+    Set to <code>true</code> to allow incoming requests to arrive on HTTP and to allow `redirect_uri` parameters to have HTTP URI addresses.
+    <br>
+    <sub>Setting this option to <code>false</code> in production is strongly encouraged to mitigate man-in-the-middle attacks.</sub>
+  </td>
+  <td><code>false</code></td>
+</tr>
+<tr>
+  <td>UseCustomViews</td>
```

Comment:
FYI: this option will be removed when https://github.com/openiddict/core/pull/25 is merged :warning: